ocs_ci/ocs/deployment_openshift_logging.py

```diff
@@ -0,0 +1,362 @@
+"""
```

Comment:
we should move this file in utils folder

Reply:
Will be done in next commit